SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -103,3 +105,13 @@ bool UGridBasedLBStrategy::IsInside(const FBox2D& Box, const FVector2D& Location
 	return Location.X >= Box.Min.X && Location.Y >= Box.Min.Y
 		&& Location.X < Box.Max.X && Location.Y < Box.Max.Y;
 }
+
+const TArray<TPair<const VirtualWorkerId*, const FBox2D*>> UGridBasedLBStrategy::GetVirtualWorkerToCell() const
```

Comment:
Slightly nervous about returning pointers to the internals.  I see in the current use case it should be ok, but if further use cases developed, callers that stored those pointers could deref invalid pointers if the internals have changed.  What do you think Ally?

Reply:
Yep, totally happy to change to copies since it's just an int / few floats